docs/tasks/deploying-functions.md

```diff
@@ -95,8 +101,7 @@ nuctl get function --namespace nuclio
 
 ```
 
-To illustrate that the function is indeed accessible via HTTP, you'll use [httpie](https://httpie.org) to invoke
-the function at the port specified by the deployment log:
+To illustrate that the function is indeed accessible via HTTP, you'll use [httpie](https://httpie.org) to invoke the function at the port specified by the deployment log:
```

Comment:
Please avoid writing long lines, or changing lines we already broke up to 1 long line like in this instance.
I think we've discussed this before, and remember you like it that way because you use line wrapping in your editor, but programmers often don't (for good reasons, usually), and it renders the same in the docs.
Since this is in-repo documentation, please keep it friendly to read/edit for us by keeping those readability-only line breaks when you see them

Reply:
Wrong, I don't like long lines at all and I've actually broken some long lines as part of my edits in this PR. I explained in the past that initially, Eran D. or Haviv (I don't recall who) didn't want to break paragraphs, but we had since then agreed to break source lines. However, I prefer not to break lines in mid sentence, which is also the convention in the doc sites. I think this is bad practice and causes problems when editing the content. If lines become too long, we should try to break the content into shorter sentences, although sometimes the source is long because of infra text that isn't visible in the output.
In this instance, I'd rather keep my modified version, unless you feel very strongly about breaking it; (you can always do it later when I'm not here to argue ...).